darshan-util/pydarshan/darshan/cli/summary.py

```diff
@@ -0,0 +1,226 @@
+import os
+import argparse
+import datetime
+import importlib.resources as importlib_resources
+from typing import Any, Union
+
+import pandas as pd
+from mako.template import Template
+
+import darshan
+import darshan.css
```

Comment:
I wonder if we really want to be adding two whole top-level namespaces to the `darshan` package when they are both thematically related to "html/template" stuff and could be organized into a single namespace at most, if not encapsulated in `cli` or whatever.

Reply:
I don't have a preference here, I ended up splitting them up but I could have gone the other way just as easily. If we want to throw them all in 1 namespace that's fine with me.

Reply:
All report-related things (`base.html`, `style.css` and `summary.py`) have been moved into `darshan.cli`